### DISCUSSION TITLE
Welcome to BlazorCodeMirror6 Discussions!

### DISCUSSION THREAD

**Discussion by gaelj** [gaelj/BlazorCodeMirror6#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pm64** [comment]
Hey friends, I've been following BlazorCodeMirror6's development for some time, and it's probably my favorite component. It's a great way to experiment with Markdown and learn about JS interop. Glad to be along for the ride and watch this community grow!
